hypothesis-python/src/hypothesis/internal/conjecture/engine.py

```diff
@@ -773,7 +783,7 @@ def reuse_existing_examples(self) -> None:
                 corpus.extend(extra)
 
             for existing in corpus:
-                data = self.cached_test_function(existing, extend=BUFFER_SIZE)
```

Comment:
ah, oops - somehow I had convinced myself that `extend=BUFFER_SIZE` was the same as not passing `extend=`. But it's not. I'll revert this.